### PR TITLE
Added installation of certificates and removed global disabling of ce…

### PR DESCRIPTION
**Pull request by MartinLundgren**
- Added installation and removal of certificate (from Windows certificate store) when a certificate is passed to CreateSecureRestMock().
- Removed global disabling of certificate validation
- Changed to find also invalid certificates in LoadCertificate()
- Bumped to version 3.0.0

### CODE REVIEW

File: src/NetMock/Rest/RestMock.cs [hunk 1]
```diff
@@ -133,6 +135,11 @@ public void Activate()
 			if (ServiceMock.ActivationStrategy == ActivationStrategy.Manual)
 				ParseRequestDefinitions();
 
+			if (Certificate != null && InstallCertificate)
+			{
+				CertificateUtil.AddCertificate(Certificate);
```
Comment (m00ndark) [inline src/NetMock/Rest/RestMock.cs:140]: Return a value indicating whether or not the certificate was installed, and use that below to only uninstall the certificate if we previously installed it.

File: src/NetMock/ServiceMock.cs [hunk 2]
```diff
@@ -108,5 +101,14 @@ public void Dispose()
 		{
 			TearDown();
 		}
+
+		private RestMock CreateSecureRestMock(string basePath, int port, X509Certificate2 certificate, bool installCertificate, MockBehavior mockBehavior = MockBehavior.Loose)
```
Comment (m00ndark) [inline src/NetMock/ServiceMock.cs:105]: Place this overload next to the others.

File: src/NetMock/Rest/RestMock.cs (hunk 1 above)
Comment (MartinLundgren) [reply src/NetMock/Rest/RestMock.cs]: Only removing the certificate if it has been previously installed by the Restmock.

File: src/NetMock/ServiceMock.cs (hunk 2 above)
Comment (MartinLundgren) [reply src/NetMock/ServiceMock.cs]: Moved.

**Review by m00ndark — APPROVED** [review]

**Pull request merged**